feature-extensions/apigee/setup.py

```diff
@@ -90,11 +95,11 @@ def install(args):
 
     # Run terraform first, as we need it's output
     runTerraform(getRegion(args), getAWSAccountID(), getEnvPrefix(args), True)
-
+    time.sleep(10)
```

Comment:
Ok, so it's an AWS API bug. Fair enough. Hopefully when we can remove the workaround when the Terraform bug is fixed it will obviate this problem.

Then can we do 2 things:

1. Use the `retrying` Python package referenced in the second link, that is a much more robust solution that isn't so dependent on a hardcoded sleep interval that may not work tomorrow.

2. Leave a comment with those links right above the spot where we do a retry so the next person will know why we had to do something suboptimal.

(Yes, I know my comment right above this about the Terraform bug doesn't reference a specific Terraform bug, I got sloppy on that and will go hunt down the actual bug # and add the comment)

Reply:
Done